File: Practice 7/Kalmykov/db-filler/src/main/kotlin/com/example/databasefiller/models/Result.kt
package com.example.databasefiller.models

import com.example.databasefiller.models.event.Event

const val MIN_RESULT = 2
const val MAX_RESULT = 250

data class Result(
    val event: Event,
    val player: Player,
    val medal: String,
    val result: Float,
)

File: Practice 7/Krymov/Hse/Hse.DbGenerator/Models/Result.cs
namespace Hse.DbGenerator.Models
{
    public class Result
    {
        public const int EventIdMaxLength = 7;
        public string EventId { get; set; }
        
        public const int PlayerIdMaxLength = 10;
        public string PlayerId { get; set; }

        public const int MedalMaxLength = 7;
        public string Medal { get; set; }
        
        public double Value { get; set; }
        
        public Player Player { get; set; }
        
        public Event Event { get; set; }
    }
}